components/scripts/lib/libs.sh

```diff
@@ -1,37 +1,40 @@
 #!/usr/bin/env bash
 
 # shellcheck source=lib/color.sh
-source "${LIB_DIR}/color.sh" || { echo "Couldn't find '${LIB_DIR}/color.sh'"; exit 1; }
+source "${LIB_DIR}/color.sh" || { echo "Couldn't find '${LIB_DIR}/color.sh'"; exit 2; }
+
+# shellcheck source=lib/exit-code.sh
+source "${LIB_DIR}/exit-code.sh" || { echo "Couldn't find '${LIB_DIR}/exit-code.sh'"; exit 2; }
 
 # shellcheck source=lib/info.sh
-source "${LIB_DIR}/info.sh" || { echo "Couldn't find '${LIB_DIR}/info.sh'"; exit 1; }
+source "${LIB_DIR}/info.sh" || { echo "Couldn't find '${LIB_DIR}/info.sh'"; exit 2; }
 
 # shellcheck source=lib/wizard.sh
-source "${LIB_DIR}/wizard.sh" || { echo "Couldn't find '${LIB_DIR}/wizard.sh'"; exit 1; }
+source "${LIB_DIR}/wizard.sh" || { echo "Couldn't find '${LIB_DIR}/wizard.sh'"; exit 2; }
 
 # shellcheck source=lib/config.sh
-source "${LIB_DIR}/config.sh" || { echo "Couldn't find '${LIB_DIR}/config.sh'"; exit 1; }
+source "${LIB_DIR}/config.sh" || { echo "Couldn't find '${LIB_DIR}/config.sh'"; exit 2; }
 
 # shellcheck source=lib/init.sh
-source "${LIB_DIR}/init.sh" || { echo "Couldn't find '${LIB_DIR}/init.sh'"; exit 1; }
+source "${LIB_DIR}/init.sh" || { echo "Couldn't find '${LIB_DIR}/init.sh'"; exit 2; }
 
 # shellcheck source=lib/git.sh
-source "${LIB_DIR}/git.sh" || { echo "Couldn't find '${LIB_DIR}/git.sh'"; exit 1; }
+source "${LIB_DIR}/git.sh" || { echo "Couldn't find '${LIB_DIR}/git.sh'"; exit 2; }
 
 # shellcheck source=lib/build_scan.sh
-source "${LIB_DIR}/build_scan.sh" || { echo "Couldn't find '${LIB_DIR}/build_scan.sh'"; exit 1; }
+source "${LIB_DIR}/build_scan.sh" || { echo "Couldn't find '${LIB_DIR}/build_scan.sh'"; exit 2; }
 
 # shellcheck source=lib/gradle.sh
-source "${LIB_DIR}/gradle.sh" || { echo "Couldn't find '${LIB_DIR}/gradle.sh'"; exit 1; }
+source "${LIB_DIR}/gradle.sh" || { echo "Couldn't find '${LIB_DIR}/gradle.sh'"; exit 2; }
 
 # shellcheck source=lib/maven.sh
-source "${LIB_DIR}/maven.sh" || { echo "Couldn't find '${LIB_DIR}/maven.sh'"; exit 1; }
+source "${LIB_DIR}/maven.sh" || { echo "Couldn't find '${LIB_DIR}/maven.sh'"; exit 2; }
 
 # shellcheck source=lib/help.sh
-source "${LIB_DIR}/help.sh" || { echo "Couldn't find '${LIB_DIR}/help.sh'"; exit 1; }
+source "${LIB_DIR}/help.sh" || { echo "Couldn't find '${LIB_DIR}/help.sh'"; exit 2; }
 
 # shellcheck source=lib/paths.sh
-source "${LIB_DIR}/paths.sh" || { echo "Couldn't find '${LIB_DIR}/paths.sh'"; exit 1; }
+source "${LIB_DIR}/paths.sh" || { echo "Couldn't find '${LIB_DIR}/paths.sh'"; exit 2; }
 
 # shellcheck source=lib/project.sh
-source "${LIB_DIR}/project.sh" || { echo "Couldn't find '${LIB_DIR}/project.sh'"; exit 1; }
+source "${LIB_DIR}/project.sh" || { echo "Couldn't find '${LIB_DIR}/project.sh'"; exit 2; }
```

Comment:
Shouldn't we return a constant instead of 2 in all these cases here and above?

Reply:
Yes, however, the constants aren't loaded until the `exit-code.sh` script is loaded.  That said, we could define a constant in this file, for this file.  That would still be more readable. I'll make that change.

Reply:
I pushed up a commit that cleans this up significantly, and also ensures that the error printed if failing to load a library is consistently formatted.